swagger.yaml

```diff
@@ -146,226 +328,821 @@ paths:
   /datasets/{id}/editions/{edition}/versions/{version}/dimensions:
     get:
       tags:
-      - "Dataset"
+      - "Public"
       summary: "Get all dimensions from a dataset"
       description: "Get all dimensions which are used in the dataset"
       parameters:
-      - $ref: '#/parameters/id'
       - $ref: '#/parameters/edition'
+      - $ref: '#/parameters/id'
       - $ref: '#/parameters/version'
       responses:
         200:
           description: "A json list of dimensions"
           schema:
             $ref: '#/definitions/Dimensions'
         404:
-          description: "No dataset edition with the specific version was found using the id, edition and version provided"
+          description: "No version was found for an edition of a dataset using the id, edition and version provided"
         500:
-          description: "Failed to process the request due to an internal error"
+          $ref: '#/responses/InternalError'
   /datasets/{id}/editions/{edition}/versions/{version}/dimensions/{dimension}:
     get:
       tags:
-      - "Dataset"
+      - "Public"
       summary: "Get all values from a dimension"
       description: "Get a list of all values which appear in this dimension and dataset"
       parameters:
-      - $ref: '#/parameters/id'
+      - $ref: '#/parameters/dimension'
       - $ref: '#/parameters/edition'
+      - $ref: '#/parameters/id'
       - $ref: '#/parameters/version'
-      - $ref: '#/parameters/dimension'
       responses:
         200:
           description: "Json object containing all values for a dimension"
           schema:
             $ref: '#/definitions/DimensionValues'
         404:
-          description: "No dataset edition with the specific version was found using the id, edition and version provided or the dimension does not exist within the dataset"
+          description: "No version was found for an edition of a dataset using the id, edition and version provided or the dimension does not exist within the dataset"
         500:
-          description: "Failed to process the request due to an internal error"
+          $ref: '#/responses/InternalError'
+  /instances:
+    get:
+      tags:
+      - "Private user"
+      summary: "Get a list of instances"
+      description: "Get a list of instances which has been paged"
+      parameters:
+        - $ref: '#/parameters/limit'
+        - $ref: '#/parameters/offset'
+        - $ref: '#/parameters/state'
+      produces:
+      - "application/json"
+      security:
+      - FlorenceAPIKey: []
+      responses:
+        200:
+          description: "Return a list of instance state"
+          schema:
+            $ref: '#/definitions/Instances'
+        500:
+          $ref: '#/responses/InternalError'
+    post:
+      tags:
+      - "Private"
+      summary: "Create an instance"
+      description: "Create an instance which will be imported"
+      parameters:
+      - $ref: '#/parameters/import_job'
```

Comment:
I think this model needs renaming as its no longer an import job?

Reply:
To create an instance we still need to maintain a link back to the import job. The job also has a link back to the instance.

Reply:
Updated the instance endpoint to take a write version of an instance model, which includes the job id and href